internal/compiler/typeloader.rs

```diff
@@ -197,16 +197,10 @@ impl TypeLoader {
                         }
                     }))
                 } else {
-                    import.file = state
-                        .borrow()
-                        .tl
-                        .resolve_import_path(
-                            Some(&import.import_uri_token.clone().into()),
-                            &import.file,
-                        )
-                        .0
-                        .to_string_lossy()
-                        .to_string();
+                    if let Some((path, _)) = state.borrow().tl.resolve_import_path(Some(&import.import_uri_token.clone().into()), &import.file) {
+                        import.file =path.to_string_lossy().to_string();
```

Comment:
```suggestion
                        import.file = path.to_string_lossy().to_string();
```

That said, I wonder if there's error handling missing though - since resolve_import_path can fail now, then maybe the `import "foo.ttf"` should fail if it was relative to the working dir?